src/Common/DomainOperationStatus.cs

```diff
@@ -26,6 +26,10 @@ public enum DomainOperationStatus
 		/// </summary>
 		Conflict,
 		/// <summary>
+		///		Client request size exceeded the file size limit set on server (gets converted to HTTP code 413 on the API)
```

Comment:
Instead of "_Client request size exceeded the file size limit set on server_" need "_the requested entity is larger than limits defined by server_".